x/group/spec/03_msg_service.md

```diff
@@ -0,0 +1,104 @@
+<!--
+order: 3
+-->
+
+# Msg Service
+
+## Msg/CreateGroup
+
+A new group can be created with the `MsgCreateGroupRequest`, which has an admin address, a list of members and some optional metadata bytes.
+
++++ https://github.com/regen-network/regen-ledger/blob/24290a0f2b219ac831482006952cb7a11d173ecf/proto/regen/group/v1alpha1/tx.proto#L53-L64
```

Comment:
Should we try the new vuepress syntax with relative paths? https://v1.vuepress.vuejs.org/guide/markdown.html#import-code-snippets

We have netlify preview, if the preview breaks on those snippets, we can fallback to permalinks

Reply:
We could try it out
Btw, this made me realize that the `x/group/spec` changes are not reflected in the VuePress app, should we move `x/group/spec/README.md` to `docs/modules/group/` while the group module is still part of regen-ledger so the new groups docs appear on docs.regen.network? cc/ @clevinson @aaronc

Reply:
hmm, would simlinking work?